test/base/MockFactory.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package base
+
+import com.codahale.metrics.Timer
+import connectors.CustomsDeclareExportsMovementsConnector
+import metrics.MovementsMetrics
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import org.scalatest.mockito.MockitoSugar
+import play.api.http.Status.INTERNAL_SERVER_ERROR
+import play.api.test.Helpers.NO_CONTENT
+import services.{CustomsCacheService, SubmissionService}
+import uk.gov.hmrc.http.HttpResponse
+import uk.gov.hmrc.http.cache.client.CacheMap
+
+import scala.concurrent.Future
+
+object MockFactory extends MockitoSugar {
+
+  private val emptyCacheMap = CacheMap("", Map.empty)
+
+  def buildCustomsCacheServiceMock: CustomsCacheService = {
+    val customsCacheServiceMock = mock[CustomsCacheService]
+    when(customsCacheServiceMock.fetchAndGetEntry(any(), any())(any(), any(), any()))
```

Comment:
Just my propose about formatting. One empty line after mock and after all whens.

It's difficult to read those code at the first time. I had problem in movements backend.

Reply:
👍